<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserEditor</class>
 <widget class="QWidget" name="UserEditor">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>191</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>User Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="username_label">
         <property name="text">
          <string>Username:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="old_pswd_label">
         <property name="text">
          <string>Old passphrase:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="new_pswd_label">
         <property name="text">
          <string>New passphrase:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="repeat_new_pswd_label">
         <property name="text">
          <string>Repeat new passphrase:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="username">
         <property name="text">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Username goes here...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="old_pswd">
         <property name="placeholderText">
          <string>Old passphrase goes here...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="new_pswd">
         <property name="placeholderText">
          <string>New passphrase goes here...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="repeat_new_pswd">
         <property name="placeholderText">
          <string>Repeat new passphrase here...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="error">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#aa0000;&quot;&gt;Error: passphrase fields differs...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="vspacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="hspacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Save</string>
       </property>
       <property name="icon">
        <iconset resource="../../picsou.qrc">
         <normaloff>:/resources/material-design/svg/check.svg</normaloff>:/resources/material-design/svg/check.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../../picsou.qrc">
         <normaloff>:/resources/material-design/svg/cancel.svg</normaloff>:/resources/material-design/svg/cancel.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../picsou.qrc"/>
 </resources>
 <connections/>
</ui>
